### ISSUE TITLE
No such file or directory .ip_address

### ISSUE THREAD

**Issue by entozoon** [ideasonpurpose/basic-wordpress-vagrant#68]
[Vagrantfile:66](https://github.com/ideasonpurpose/basic-wordpress-vagrant/blob/9512a036d8fd07a68d4e231e2edc65e7ee5c4269/Vagrantfile#L66) is complaining this file doesn't exist. 

Do I create that file myself? Possibly containing the intended IP for the machine?

**Comment by entozoon** [comment]
Hmm, actually I can't replicate it. Must have had a borked VBox VM instance, my apologies!

**Comment by joemaller** [comment]
Actually I just saw something similar on another vagrant project and this helped me figure out a better solution over there. 

That `/vagrant/.ip_address` file was a bit of a hack and way too fragile. The file got written as a side-effect deep in the wp_config task here: https://github.com/ideasonpurpose/basic-wordpress-vagrant/blob/8409cc67ac8128110f7b75fb20dafecc5e610084/ansible/roles/wordpress/tasks/wp-config.yml#L3-L5

It would be nice to store the IP address in a variable in the VagrantFile, but I couldn't figure out how to hook the Vagrant:Machine instance outside of the plugin block. So instead, I added a one-line shell provisioner which explicitly writes the .ip_address file.

It's a lot better over here:  https://github.com/ideasonpurpose/basic-wordpress-vagrant/blob/ba0af714a6e1be86e52a164e9501dc4dead4a2e6/Vagrantfile#L107-L109

I've also been having issues with Apache failing to restart, so I added a one-liner to force-kick that too. 

Thanks for reporting, even if it was just a glitch 👍
